modules/ossm-auto-route.adoc

```diff
@@ -43,25 +42,3 @@ gateway1-scqhv www.bookinfo.com            istio-ingressgateway   <all>
 ----
 
 If the gateway is deleted, {ProductName} deletes the routes. However, routes created manually are never modified by {ProductName}.
-
-[id="ossm-auto-route-annotations_{context}"]
-= Annotations
```

Comment:
I am looking in 4.9 and 4.10 and I don't actually see this annotations section in there. Only in main and 4.8 and earlier. 

@JStickler - I'm thinking that #34854 was supposed to have gotten cherry picked to 4.9 (and now also 4.10). Can you PTAL?

Reply:
@kalexand-rh Do I need to do that before you can merge this PR?  (Just trying to figure out timing here...)

Reply:
@JStickler, it would be a little easier if you did!

Reply:
#34854 Cherry picked to 4.9 and 4.10/